dev/dns.go

```diff
@@ -2,16 +2,27 @@ package dev
 
 import (
 	"net"
-	"sync"
 	"time"
 
 	"github.com/miekg/dns"
+	"gopkg.in/tomb.v2"
 )
 
-const DefaultAddress = ":9253"
-
 type DNSResponder struct {
 	Address string
+	Domains []string
```

Comment:
now a single `DNSResponder` instance is bound to its domains, rather than being able to call `Serve` multiple times with diff argumnets on the same instance.